README.md

```diff
@@ -59,8 +60,7 @@ Similar approach for configuration, here we want to login as Keycloak user, usin
 The project uses [aerogear-ios-http](https://github.com/aerogear/aerogear-ios-http) framework for its http calls, and [aerogear-ios-httpstub](https://github.com/aerogear/aerogear-ios-httpstub) framework for stubbing its http network requests.  To handle these dependencies, it uses [Cocoapods](http://cocoapods.org). On the root directory of the project run:
 
 ```bash
-bundle install
-bundle exec pod install
+pod install
```

Comment:
let remove the 'Drag the library in your project'  and reuse [the instructions](https://github.com/corinnekrych/aerogear-ios-httpstub/blob/cocopods.0.36.support/README.md) we have already in the http-lib for the "Manual Installation"